.github/workflows/release-nightly.yml

```diff
@@ -122,10 +121,10 @@ jobs:
       GH_ORG: ${{ secrets.GH_ORG }}
       GH_TOKEN: ${{ secrets.GH_TOKEN }}
 
-  publish-releases:
+  publish-gh-releases:
     name: Publish releases
     needs: [keycloak, keycloak-documentation, keycloak-nodejs-admin]
-    uses: ./.github/workflows/x-publish-releases.yml
+    uses: ./.github/workflows/x-publish-gh-releases.yml
     with:
       tag: nightly
     secrets:
```

Comment:
After `secrets` in release.yml, you've added the new section `publish-npm`. Is it intentionally left out of the nightly workflow here? i.e. NPM nightlies will be available from the releases tab only?

Reply:
Yes, that's correct. We do not publish nightlies through NPM as it is not possible to publish there without a unique version number for each published package (e.g. `19.0.0-dev.1`, `19.0.0-dev.2`, etc). It is also not possible to publish over an existing version, which makes it hard to do snapshots properly.

We will ensure that the correct versions are used by downloading the tarballs from GH releases and installing them directly locally (`npm i admin-client.tgz`). This will happen in a later PR.